### ISSUE TITLE
Incompatible with Django 1.6 custom Users

### ISSUE THREAD

**Issue by johngoodleaf** [peterldowns/djoauth2#7]
I have a custom User. On syncdb with djoauth2 0.5, the errors returned are:

```
CommandError: One or more models did not validate:
djoauth2.client: 'user' defines a relation with the model 'auth.User', which has been swapped out. Update the relation to point at settings.AUTH_USER_MODEL.
djoauth2.authorizationcode: 'user' defines a relation with the model 'auth.User', which has been swapped out. Update the relation to point at settings.AUTH_USER_MODEL.
djoauth2.accesstoken: 'user' defines a relation with the model 'auth.User', which has been swapped out. Update the relation to point at settings.AUTH_USER_MODEL.
```

**Comment by peterldowns** [comment]
Definitely an issue! Thanks for the pull request, I'd love to get this fixed. I think the only tricky part might be doing it in such a way that the code remains compatible with 1.4 / 1.5, and also in such a way that the migration path isn't messed up.

**Comment by peterldowns** [comment]
Closed by #9.
